admin/init.sls

```diff
@@ -27,9 +27,36 @@ UTC:
     - mode: 644
     - source: salt://{{ tpldir }}/files/hosts
 
-{% for ssh_user in admin.ssh_users %}
-sshkey-{{ ssh_user }}:
-  ssh_auth.present:
+# FIXME we should also explicitly AllowUsers usera userb userc
```

Comment:
We could also use `AllowGroups wheel`, I think.

Reply:
I believe we want to end up with as few people having root access as possible, so most users who can access the hosts to examine logs or whatever will not be in `wheel`. Thus they'll either need to be individually added with `AllowUsers`, or all added to a group which in turn is added using `AllowGroups`. While a dedicated group would have the same effect as individually allowing, it adds a layer of indirection when trying to figure out which users have access compared to just looking at `sshd_config`. 

So my FIXME was pretty unclear, but I'm pretty sure it's an issue that `AllowGroups wheel` does not adequately address.